src/init.c

```diff
@@ -1263,11 +1263,17 @@ void jl_get_builtin_hooks(void)
     jl_interrupt_exception = jl_new_struct_uninit((jl_datatype_t*)core("InterruptException"));
     jl_boundserror_type    = (jl_datatype_t*)core("BoundsError");
     jl_memory_exception    = jl_new_struct_uninit((jl_datatype_t*)core("MemoryError"));
+    #ifdef _OS_LINUX_
+    #ifdef _P32
+        jl_float64_type->alignment = 4;
```

Comment:
@Keno why is this here? https://github.com/JuliaLang/julia/commit/4a697d145ab719c11b019264d9f579c48b03b258

MAX_ALIGN should be 4 anyways on `_OS_LINUX_` with `_P32`